### ISSUE TITLE
How can I copy a model trained by one machine to another PC and make it worked ?

### ISSUE THREAD

**Issue by skyzhao3q** [NVIDIA/DIGITS#960]

**Comment by IsaacYangSLA** [comment]
@skyzhao3q , you can first download the trained weights to you local file system.  Copy that file to another machine.  On that new machine, untar/unzip trained weights file to get one weight file (caffemodel).  Run DIGITS on new machine and create a new model.  In the new model page, upload the caffemodel as pretrained weight.  You might need to set other parameters, like learning rate, close to the value used in the final moment of the previous training.  This is usually working well.  If you have other question, can you describe it in more details?  Thanks.
